assets/js/admin/course-theme/guide.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { LEFT, RIGHT } from '@wordpress/keycodes';
+import { Modal, Button } from '@wordpress/components';
+
+/**
+ * This component is an adaptation of Guide component from Gutenberg.
+ * It was adapted mainly to allow an action when closing the onboarding,
+ * and different ones when clicking on the buttons.
+ *
+ * @link https://github.com/WordPress/gutenberg/tree/trunk/packages/components/src/guide
```

Comment:
Couldn't believe the Gutenberg's `Guide` component is not flexible enough so we could use it. Went to look into it's api and source code and it turns out it isn't flexible and we have to come up with our own. Well 🤷

Reply:
Ah, good catch! Removed that in this commit: https://github.com/Automattic/sensei/pull/4599/commits/3454ca71cc40a3f79bb94bd6076ab924ce2b837b